doc/03_reference/images/tb2.svg

```diff
@@ -0,0 +1,989 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The figure is hardly readable in the html. If you have time, you can maybe reduce the left and right borders as well as the empty space on the right inside the violet box.